src/payment/checkout/payment-form/CardDetails.jsx

```diff
@@ -209,7 +213,16 @@ export class CardDetailsComponent extends React.Component {
           />
         </h5>
 
-        <div className="row">
+        {this.props.flexMicroformEnabled && <FlexMicroform
+          captureKeyId={this.props.captureKeyId}
+          render={
+            ({ microform }) => (
```

Comment:
I think this may be a mis-application of render props, which seem to be about cross-cutting concerns: https://reactjs.org/docs/render-props.html

Reply:
I'm not sure if this can just be `props.children` based?

Reply:
Yeah, I'm not sure render props is buying you much here.  Might be easier to read to just include the <div> below in the actual <FlexMicroform> component's render function directly.  

You might use render props if you needed to render FlexMicroform with different children in multiple versions of a <CardDetails> component, and those children needed access to a set of props/state defined in FlexMicroform.  The second half of that is what's happening, but we don't seem to have the "different children in multiple places" part.  Lemme know if this isn't making sense.

Reply:
Makes sense, I think we can just move them into the Render function for the form!